solidity/contracts/system/TBTCSystem.sol

```diff
@@ -151,16 +154,16 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
         return allowNewDeposits;
     }
 
+    /// @notice Return the lowest lot size currently enabled for deposits.
+    /// @return The lowest lot size, in satoshis.
+    function getMinLotSize() public view returns (uint256) {
```

Comment:
Hmmm… Now that I see this I kind of feel like we do `Minimum` and `Maximum` here... Thoughts?

Reply:
Yeah... I did `Min` to keep it consistent with the existing `Max` function but I have the same feelings about `Minimum` and `Maximum`. I'll update it.